src/main/java/hudson/plugins/sauce_ondemand/SauceOnDemandReportPublisher.java

```diff
@@ -281,7 +281,7 @@ private void processBuildOutput(Run build, SauceOnDemandBuildAction buildAction,
     }
 
     protected SauceREST getSauceREST(Run build) {
-        SauceCredentials credentials = build.getAction(SauceOnDemandBuildAction.class).getCredentials();
+        SauceCredentials credentials = getBuildAction(build).getCredentials();
```

Comment:
The function is SauceOnDemandBuildAction, so this won't work will it?

Does anywhere look good to share it?